crypto-app/src/hooks/useMediaQuery.ts

```diff
@@ -0,0 +1,19 @@
+import { useState, useEffect } from "react";
+
+const useMediaQuery = (query: string) => {
+  const [matches, setMatches] = useState(false);
+
+  useEffect(() => {
+    const media = window.matchMedia(query);
```

Comment:
Before doing this check if query even exists/is not empty

`if(!query) return`